vignettes/fims-path-maturity.Rmd

```diff
@@ -0,0 +1,549 @@
+---
+title: "FIMS Path: Maturity"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{FIMS Path: Maturity}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+This vignette describes the hierarchical structure of FIMS by describing the linkages, or *path* through the FIMS model using maturity as an example. The vignette walks through how input from R is carried into the core FIMS calculations. 
+
+## Setting up a module in R and linking to the Rcpp interface
```

Comment:
I got a little lost between code that you have to write yourself (in R in a script) to create the maturity module and then the code that is already written as part of FIMS. I suggest putting everything that a user has to write in a single section up top (including how to clear FIMS), and then adding a clear heading to signal transition to talking about the code that is within FIMS and the user does not have to write.

Reply:
I attempted to color code the code chunks but think this might be too much. I  would prefer to show you the colored version and get your feedback before pushing up that change!